<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>PlayerWindow</class>
    <widget class="QWidget" name="PlayerWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>240</width>
                <height>320</height>
            </rect>
        </property>

        <property name="styleSheet">
            <string notr="true">
                QWidget#PlayerWindow {
                background-color: #121212;
                border-radius: 10px;
                }

                QLabel {
                color: #ffffff;
                background-color: transparent;
                }

                QLabel#lblCover {
                background-color: #1e1e1e;
                border-radius: 8px;
                border: 2px solid #2a2a2a;
                }

                QLabel#lblTitle {
                font-size: 14px;
                font-weight: 600;
                }

                QLabel#lblArtist {
                color: #b3b3b3;
                font-size: 12px;
                }

                QProgressBar {
                background-color: #2a2a2a;
                border: none;
                border-radius: 4px;
                height: 6px;
                }

                QProgressBar::chunk {
                background-color: #1db954;
                border-radius: 4px;
                }

                QPushButton {
                background-color: #1e1e1e;
                color: #ffffff;
                border: 1px solid #2a2a2a;
                border-radius: 8px;
                padding: 6px 10px;
                font-size: 13px;
                min-width: 40px;
                min-height: 36px;
                }

                QPushButton#btnPlay {
                background-color: #1db954;
                border-color: #1db954;
                font-weight: 600;
                min-width: 60px;
                }
            </string>
        </property>

        <layout class="QVBoxLayout" name="vLay">
            <property name="spacing"><number>8</number></property>
            <property name="leftMargin"><number>10</number></property>
            <property name="topMargin"><number>10</number></property>
            <property name="rightMargin"><number>10</number></property>
            <property name="bottomMargin"><number>10</number></property>

            <!-- ОБЛОЖКА ПО ЦЕНТРУ -->
            <item>
                <layout class="QHBoxLayout" name="hLayCover">
                    <property name="alignment"><set>Qt::AlignHCenter</set></property>
                    <item>
                        <widget class="QLabel" name="lblCover">
                            <property name="minimumSize"><size><width>160</width><height>160</height></size></property>
                            <property name="maximumSize"><size><width>160</width><height>160</height></size></property>
                            <property name="alignment"><set>Qt::AlignCenter</set></property>
                            <property name="scaledContents"><bool>true</bool></property>
                            <property name="text"><string>Обложка</string></property>
                        </widget>
                    </item>
                </layout>
            </item>

            <item>
                <widget class="QLabel" name="lblTitle">
                    <property name="alignment"><set>Qt::AlignCenter</set></property>
                    <property name="text"><string>Название трека</string></property>
                </widget>
            </item>

            <item>
                <widget class="QLabel" name="lblArtist">
                    <property name="alignment"><set>Qt::AlignCenter</set></property>
                    <property name="text"><string>Исполнитель</string></property>
                </widget>
            </item>

            <item>
                <widget class="QProgressBar" name="pb">
                    <property name="value"><number>0</number></property>
                    <property name="textVisible"><bool>false</bool></property>
                </widget>
            </item>

            <!-- КНОПКИ -->
            <item>
                <layout class="QHBoxLayout" name="hLayButtons">
                    <property name="spacing"><number>8</number></property>

                    <item>
                        <widget class="QPushButton" name="btnPrev">
                            <property name="text"><string>⏮</string></property>
                        </widget>
                    </item>

                    <item>
                        <spacer name="spLeft">
                            <property name="orientation"><enum>Qt::Horizontal</enum></property>
                            <property name="sizeHint" stdset="0"><size><width>10</width><height>20</height></size></property>
                        </spacer>
                    </item>

                    <item>
                        <widget class="QPushButton" name="btnPlay">
                            <property name="text"><string>▶</string></property>
                        </widget>
                    </item>

                    <item>
                        <spacer name="spRight">
                            <property name="orientation"><enum>Qt::Horizontal</enum></property>
                            <property name="sizeHint" stdset="0"><size><width>10</width><height>20</height></size></property>
                        </spacer>
                    </item>

                    <item>
                        <widget class="QPushButton" name="btnNext">
                            <property name="text"><string>⏭</string></property>
                        </widget>
                    </item>

                </layout>
            </item>

        </layout>
    </widget>

    <resources/>
    <connections/>
</ui>
